<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FacilityInfo2</class>
 <widget class="QDialog" name="FacilityInfo2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Facility Info</string>
  </property>
  <widget class="QPushButton" name="d01_button">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>160</y>
     <width>200</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0;
background-color: #52b788;</string>
   </property>
   <property name="text">
    <string>D01</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>630</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15;</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="c04_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>225</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0;
background-color: #52b788;</string>
   </property>
   <property name="text">
    <string>C04</string>
   </property>
  </widget>
  <widget class="QPushButton" name="d02_button">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>160</y>
     <width>200</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0;
background-color: #52b788;</string>
   </property>
   <property name="text">
    <string>D02</string>
   </property>
  </widget>
  <widget class="QPushButton" name="c05_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>300</y>
     <width>225</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0;
background-color: #52b788;</string>
   </property>
   <property name="text">
    <string>C05</string>
   </property>
  </widget>
  <widget class="QPushButton" name="c01_button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>225</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0;
background-color: #52b788;</string>
   </property>
   <property name="text">
    <string>C01</string>
   </property>
  </widget>
  <widget class="QLabel" name="bg_white">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>1000</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #f2e9e4;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="c02_button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>225</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0;
background-color: #52b788;</string>
   </property>
   <property name="text">
    <string>C02</string>
   </property>
  </widget>
  <widget class="QLabel" name="bg_dark">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #22223b;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="c03_button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>225</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0;
background-color: #52b788;</string>
   </property>
   <property name="text">
    <string>C03</string>
   </property>
  </widget>
  <widget class="QPushButton" name="c06_button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>450</y>
     <width>225</width>
     <height>125</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0;
background-color: #52b788;</string>
   </property>
   <property name="text">
    <string>C06</string>
   </property>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>-10</y>
     <width>481</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #fff;</string>
   </property>
   <property name="text">
    <string>Facility Info 2F</string>
   </property>
  </widget>
  <zorder>bg_dark</zorder>
  <zorder>bg_white</zorder>
  <zorder>d01_button</zorder>
  <zorder>backButton</zorder>
  <zorder>c04_button</zorder>
  <zorder>d02_button</zorder>
  <zorder>c05_button</zorder>
  <zorder>c01_button</zorder>
  <zorder>c02_button</zorder>
  <zorder>c03_button</zorder>
  <zorder>c06_button</zorder>
  <zorder>title_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
